tests/fips/fips_setup.pm

```diff
@@ -33,76 +41,38 @@ sub run {
         assert_script_run("grep '^GRUB_CMDLINE_LINUX_DEFAULT.*fips=1' /etc/default/grub");
         assert_script_run("grep '^1\$' /proc/sys/crypto/fips_enabled");
         record_info 'Kernel Mode', 'FIPS kernel mode (for global) configured!';
-
```

Comment:
this looks informative, why to remove it? I'm thinking in a bug of the kind that fips is setup but the package is removed somehow, but unlikely, perhaps more likely if the pattern do not install all the package needed. wdyt?

Reply:
IMHO, I don't think this check is very useful because:
1. if the system correctly boots in FIPS mode, then all packages are installed
2. that code is only doing a zypper search without checking the exit code or comparing against some text. If I'm not missing something, I don't think it's anything useful right now